network/src/test/client.rs

```diff
@@ -122,7 +117,8 @@ impl Api for TestApi {
 
 impl TestApi {
     fn add_node(&self, node: NodeId) {
```

Comment:
Since negotiation api is removed, `add_node` needs to be merged with `connected` and `allow/deny_connection` function.